Apps/UnitTests/UnitTests.cpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "gtest/gtest.h"
+#include <UrlLib/UrlLib.h>
+#include <arcana/threading/task.h>
+
+namespace Babylon
+{
+    void WaitForCompletion(arcana::task<void, std::exception_ptr> task) {
+        bool done{false};
+        task.then(arcana::inline_scheduler, arcana::cancellation::none(), [&done](arcana::expected<void, std::exception_ptr> result) {
+            done = true;
+        });
+        while (!done);
```

Comment:
This is a temporary hack. Might be good to have a comment on this.

Reply:
Done!